docker-compose.yml

```diff
@@ -1,12 +1,9 @@
-version: '3.8'
+version: "3.8"
 services:
-
   plex:
     image: linuxserver/plex:${PLEX_VERSION}
     container_name: plex
-    depends_on:
-      - protonvpn
-    network_mode: service:protonvpn
+    network_mode: host
```

Comment:
thats what I changed as well. Makes sense to have this on the host network

Reply:
@clemone210 

In the new release (just published it today) I put it together with all the other containers as they can communicate in their "internal network" as localhost/127.0.0.1 .

Please let me know if the new version works and solves all the problems, lots of changes there.

Reply:
Reverted to host in PR #16